pygac/__init__.py

```diff
@@ -35,3 +35,12 @@
 if not os.path.exists(CONFIG_FILE) or not os.path.isfile(CONFIG_FILE):
     LOG.warning(str(CONFIG_FILE) + " pointed to by the environment "
                 + "variable PYGAC_CONFIG_FILE is not a file or does not exist!")
+
+def get_absolute_azimuth_angle_diff(sat_azi, sun_azi):
+    """ Calculates absolute azimuth difference angle from
+    Satellite zenith angle and sun zenith angle """
```

Comment:
The first line of docstrings should start with a capital letter (not space), finish with a period (.) and fit on one line.